python/ml4ir/base/data/tfrecord_helper.py

```diff
@@ -45,13 +46,17 @@ def get_example_proto(row, features):
     for feature_info in features:
         feature_name = feature_info["name"]
         feature_fn = _get_feature_fn(feature_info["dtype"])
+        # FIXME
         # When applying functions with axis=1, pandas performs upcasting,
         # so if we have a mix of floats/ints, converts everything to float
         # that breaks this part of the code. Example:
         # https://stackoverflow.com/questions/47143631/
         # how-do-i-preserve-datatype-when-using-apply-row-wise-in-pandas-dataframe
-        value_upcasted = row[feature_name].astype(feature_info["dtype"])
```

Comment:
This fix doesn't really work because `row[feature_name]` is a scalar and not a pandas dataframe or series. It is not an issue for now. But I've added a `FIXME`